classes/class3/homework/views/newcat.handlebars

```diff
@@ -0,0 +1,6 @@
+<h1>Say Hello to:</h1>
+<ul>
+  <li>Name: {{name}}</li>
+  <li>Age: {{age}}</li>
+  <li>Colors: {{#each colors}}{{this}}, {{/each}}</li>
```

Comment:
Trailing comma would be nice to remove (look at `unless` and `@last` handlebars syntax)